content/en/docs/ops/configuration/security/security-policy-examples/index.md

```diff
@@ -57,11 +57,19 @@ spec:
 
 ### Namespace isolation
 
-The following policy allows traffic only from the namespace `foo`. In other words, it isolates the namespace `foo` from other
-namespaces. This requires you have already enabled mTLS.
+The following two policies enable strict mTLS and allow traffic from the namespace `foo`.
```

Comment:
enable strict mTLS on namespace 'foo', and allow traffic from the same namespace.